hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -138,14 +154,11 @@ def queries(query_group,
             else:
                return None
 
-
     orig_filename = query_file
-    query_file = __salt__['cp.cache_file'](query_file)
+
     if query_file is None:
         log.error('Could not find file {0}.'.format(orig_filename))
         return None
-    with open(query_file, 'r') as fh:
-        query_data = yaml.safe_load(fh)
 
     if not isinstance(query_data, dict):
         raise CommandExecutionError('Query data is not formed as a dict {0}'
```

Comment:
Another instance of error handling that we can move up into your for loop. We should probably check if a given file's data is formed as a dict before we attempt to merge it with our query_data.